src/test/java/org/fundacionjala/coding/Movies/TestCustomer.java

```diff
@@ -0,0 +1,188 @@
+    }
+
+    /**
+     * .
+     */
+    @Test
+    public void testCustomerRentalMovieBatmanFor2Days() {
+        Customer customerTest = new Customer("test");
+        Movie movie = MovieFactory.createRegularMovie(REGULAR_MOVIE_TITLE);
+        customerTest.rentMovie(movie, DAYS_RENT_2);
+        assertEquals("Batman", customerTest.getRentedMoviesList().get(0).getMovie().getTitle());
+        assertEquals(DAYS_RENT_2, customerTest.getRentedMoviesList().get(0).getDaysRented());
+    }
+
+    /**
+     * .
+     */
+    @Test
+    public void testCustomerGetRentMoviesTitles() {
+        Customer customerTest = new Customer("test");
+        Movie movie1 = MovieFactory.createRegularMovie(REGULAR_MOVIE_TITLE);
+        customerTest.rentMovie(movie1, DAYS_RENT_1);
+        Movie movie2 = MovieFactory.createRegularMovie("Superman");
+        customerTest.rentMovie(movie2, DAYS_RENT_1);
+        Movie movie3 = MovieFactory.createRegularMovie("Iron Man");
+        customerTest.rentMovie(movie3, DAYS_RENT_1);
+        assertEquals("Batman", customerTest.getRentedMoviesList().get(0).getMovie().getTitle());
+        assertEquals("Superman", customerTest.getRentedMoviesList().get(1).getMovie().getTitle());
+        assertEquals("Iron Man", customerTest.getRentedMoviesList().get(2).getMovie().getTitle());
+    }
+
+    /**
+     * .
+     */
+    @Test
+    public void testRent2DaysGetRentAmountForRegularMovie() {
+        Customer customerTest = new Customer("juan");
+        Movie regularMovie = MovieFactory.createRegularMovie("Batman");
+        customerTest.rentMovie(regularMovie, DAYS_RENT_2);
+        //rent regular: 2 per day, if(daysRented>2)--> 1.5 per day;
+        //(2*2) = 4
+        assertEquals(RENT_AMOUNT_4, customerTest.getTotalBillAmount());
+    }
+
+    /**
+     * .
+     */
+    @Test
+    public void testRent6DaysGetRentAmountForRegularMovie() {
+        Customer customerTest = new Customer("juan");
+        Movie regularMovie = MovieFactory.createRegularMovie("Batman");
+        customerTest.rentMovie(regularMovie, DAYS_RENT_6);
+        //rent regular: 2 per day, if(daysRented>2)-->  1.5 per day;
+        //(2*2) = 4 + (4*1.5) = 10
+        assertEquals(RENT_AMOUNT_10, customerTest.getTotalBillAmount());
+    }
+
+    /**
+     * .
+     */
+    @Test
+    public void test5DaysGetRentAmountForNewReleaseMovie() {
+        Customer customerTest = new Customer("juan");
+        Movie newMovie = MovieFactory.createNewReleaseMovie("Batman");
+        //rent new release: DaysRented * 3 ;
+        customerTest.rentMovie(newMovie, DAYS_RENT_5);
+        //(5*3) = 15
+        assertEquals(RENT_AMOUNT_15, customerTest.getTotalBillAmount());
+    }
+
+    /**
+     * .
+     */
+    @Test
+    public void test10DaysGetRentAmountForChildrenMovie() {
+        Customer customerTest = new Customer("juan");
+        Movie childMovie = MovieFactory.createChildrenMovie(CHILDREN_MOVIE_TITLE);
+        //rent children: 1.5 per day;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)